data/formats-data.js

```diff
@@ -6108,7 +6108,7 @@ exports.BattleFormatsData = {
 		tier: "New",
 	},
 	tapulele: {
-		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "aromatherapy"],
+		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "taunt", "psychic"],
```

Comment:
Still not a fan of Psychic. Psyshock is better, especially with CM. Not to mention, you can avoid the whole Psychic + Psyshock on the same set.